src/vector/platform/VectorBasePlatform.js

```diff
@@ -18,6 +18,8 @@ limitations under the License.
 */
 
 import BasePlatform from 'matrix-react-sdk/lib/BasePlatform';
+import _t from 'matrix-react-sdk/lib/languageHandler';
```

Comment:
Yep, typically I had only tested this on `WebPlatform` which was the only one to import correctly. Thanks.